website/versioned_docs/version-2.x/getting-started/create-your-first-application.md

```diff
@@ -0,0 +1,245 @@
+---
+sidebar_position: 2
+sidebar_label: Quickstart
+---
+
+
+# Quickstart: Create your first application with KafkaFlow
+
+In this article, you use C# and the .NET CLI to create two applications that will produce and consume events from Apache Kafka.
+
+By the end of the article, you will know how to use KafkaFlow to either Produce or Consume events from Apache Kafka.
+
+
+## Prerequisites
+
+ - [.NET 6.0 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/6.0)
+ - [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+
+## Overview
+
+You will create two applications:
+
+ 1. **Consumer:** Will be running waiting for incoming messages and will write them to the console.
+ 2. **Producer:** Will send a message every time you run the application.
+
+To connect them, you will be running an Apache Kafka cluster using Docker.
+
+## Steps
+
+### 1. Create a folder for your applications
+
+Create a new folder with the name _KafkaFlowQuickstart_.
+
+### 2. Setup Apache Kafka
+
+Inside the folder from step 1, create a `docker-compose.yml` file. You can download it from [here](../../../docker-compose.yml).
```

Comment:
```suggestion
Inside the folder from step 1, create a `docker-compose.yml` file. You can download it from [here](../../../../docker-compose.yml).
```

Reply:
Absolute link added, since it is referencing an external resource to the documentation its better to have the absolute link. Please check